myuw/dao/registration.py

```diff
@@ -2,6 +2,7 @@
 This module provides access to uw_sws registration module
 """
 
+from copy import deepcopy
```

Comment:
Why is deepcopy necessary here?

Reply:
Oh, because you're altering a list you're iterating through?

Reply:
Because removing the list item will mess up the list (some item may be skipped)

Reply:
I think you might want copy() there instead, since you aren't modifying the instructor objects, merely the list